packages/universal-test-adapter-dotnet/src/index.ts

```diff
@@ -47,14 +50,33 @@ export async function executeTests({ testsToRun = [] }: AdapterInput): Promise<A
     }
     return `(FullyQualifiedName~${parseFilepathAndClassName(filepath, suiteName)}.${testName})`
   })
+
   if (fullyQualifiedNames.length > 0) {
     args.push(`--filter`)
     args.push(`${fullyQualifiedNames.join(' | ')}`)
   }
+
+  // https://github.com/Microsoft/vstest-docs/blob/main/docs/report.md#syntax-of-default-loggers
+  switch (reportFormat) {
+    case 'default':
+      // Push trx logger argument so dotnet generates a trx report
+      args.push('--logger', `trx;LogFileName=${path.join(process.cwd(), 'results.trx')}`)
```

Comment:
I'll make a follow-up to pass cwd as an argument to some sort of context object, since it will make testing easier and writing new adapters easier. I imagine cwd will be a relatively common need and adapters shouldn't have to guess if it's safe to just use process.cwd(), and it gives us more flexibility for changing how adapters are executed in the future. Will require a small change to the adapter interface